ui/components/multichain/pages/review-permissions-page/site-cell/site-cell-connection-list-item.js

```diff
@@ -51,7 +51,7 @@ export const SiteCellConnectionListItem = ({
         iconName={iconName}
         size={AvatarIconSize.Md}
         color={IconColor.iconAlternative}
-        backgroundColor={BackgroundColor.backgroundMuted}
```

Comment:
I feel like avatars should remain muted

Reply:
Good suggestion. Updated https://github.com/MetaMask/metamask-extension/pull/33764/commits/e1e76b7cda36e18c5353e5f1a7bf147c293fbb12